voight_kampff.gemspec

```diff
@@ -21,9 +21,9 @@ Gem::Specification.new do |s|
   s.test_files    = `git ls-files -- {tests}/**/*`.split("\n")
   s.require_path  = 'lib'
 
-  s.add_dependency 'rack', ['>= 1.4', '< 3.0']
+  s.add_dependency 'rack', ['>= 1.4']
```

Comment:
JFYI, it's dangerous. It'd be better to leave the version lock, but upper. Like `< 4.0`.